### PR TITLE
[DPP-1050] [DPP-1029] Fix input focus and alignment problem

### PR DESCRIPTION
**Pull request by sabs231**
## What did you implement:

Implementing issue: [DPP-1050](https://fng-jira.fox.com/browse/DPP-1050)
Implementing issue: [DPP-1029](https://fng-jira.fox.com/browse/DPP-1029)

## How did you implement it:
Us the `autofocus` prop for inputs when the user selects the `unique` select
Fix selected item alignment problem so it looks like the mocks

## Files to review:
Please review the following files and ignore the rest because they are auto generated by `nps`
1. `less/control.less`
1. `src/Option.js`
1. `src/Select.js`

## How can we verify it:
1. On the DPP `package.json` file replace the `@foxdcg/react-select` dependency with this:

```json
"@foxdcg/react-select": "git://github.com/foxbroadcasting/react-select.git#bug/DPP-1050",
```

1. Run `npm i && npm start`
1. Open the DPP on the browser
1. Click on the `+` to open Global Creation flow
1. Select `New Query`
1. Select `Video Queries`
1. Select `None` (it can be any type of Query)
1. Select a panel from the `Assign To Panel` input on the top right corner of the Create a new Video Query screen
1. Notice the `focus` is moved to the search input.
1. Search for `FX Presents` and select it
1. Click outside of the Select component and again on the select component
1. Look for `FX Presents` again
1. Notice the selected option is aligned as in the gallery [mocks](https://gallery.io/projects/MCHbtQVoQ2HCZfNvzPdzBSok/files/MCHtA7U1iMGr67g5L1tTjT-XhkieAxJkg6o)
1. Profit 💰 

## Screenshots
mnk1ieorch

## Related PRs
 

- Type Settings PR: N/A 
- API PR: N/A 

## Checklist

- [ ] I added the necessary documentation, if appropriate.
- [ ] I added tests to prove that my fix is effective or my feature works.
- [ ] I reviewed existing Pull Requests before submitting mine.

### CODE REVIEW

**Comment by ragrum** [comment]
@sabs231 when you say:

> Notice the selected option is aligned as in the gallery mocks.

What's the aligment being corrected?

I can see this in your change:

image

But in the linked mocks I see:

image

**Review by ragrum — APPROVED** [review]
LGTM 👍

**Review by aarregui — APPROVED** [review]

File: less/control.less [hunk 1]
```diff
@@ -158,10 +170,10 @@
 
   .Select-option {
     &.is-selected {
-      display: inline-block;
       background-color: initial;
-      padding: 0;
     }
+
+    padding: 8px 0 8px 22px;
```
Comment (omar-aguilar) [inline less/control.less]: Please make this change only for the unique option

| Dev | Local |
| --- | --- |
| dev | local |

Comment (omar-aguilar) [reply less/control.less]: same as the above, but where you removed the `inline-block` and `padding`

| Dev | local |
| --- | --- |
| local | dev |

**Review by omar-aguilar — APPROVED** [review]

**Pull request merged**